app/src/main/java/org/ole/planet/myplanet/ui/sync/SyncActivity.kt

```diff
@@ -606,6 +613,16 @@ abstract class SyncActivity : ProcessUserDataActivity(), SyncListener, CheckVers
                         if (position > 0) {
                             val selectedTeam = teams[position - 1]
                             selectedTeamId = selectedTeam._id
+                            prefData.setSELECTEDTEAMID(selectedTeamId)
+                            val url = "${settings.getString("serverProtocol", "")}${serverUrl.text}"
+                            if (isUrlValid(url)) {
+                                if (this@SyncActivity is LoginActivity) {
+                                    this@SyncActivity.getTeamMembers()
+                                }
```

Comment:
only these 3 lines are necessary for this functionality.
I have noticed only these buttons have the functionality to dismiss the dialog
```
.positiveText
.negativeText
.neutralText
.onPositive
.onNeutral
```
so we need to figure a way of passing the dialog to this click listener to be able to dismiss